docs/install/installer.rst

```diff
@@ -0,0 +1,13 @@
+=============================
+Quick Start Flocker Installer
+=============================
+
+If you want to get started with Flocker quickly, but in your own environment, you can use the Installer.
```

Comment:
Sorry that my comments are so wishy washy, but this "looks weird (TM)". "the Installer" > "the Quick Start Flocker Installer"?